### ISSUE TITLE
/f map -- ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by Write** [MassiveCraft/Factions#810]
Error when calling /f map. It seems to happen only after chunk were claimed. On terrain they are only few claims it seems to work. 

http://pastebin.com/XcX9rEGt

**Comment by oloflarsson** [comment]
In that area; are there loads of different factions nearby? Say about 65 of them or more?

**Comment by Write** [comment]
Yup, exactly that.

**Comment by Write** [comment]
Update ?

**Comment by Write** [comment]
... ?

**Comment by Write** [comment]
:/

**Comment by Write** [comment]
Stillll waitiiiing...

**Comment by MarkehMe** [comment]
Are you still getting this issue?

**Comment by magnusulf** [comment]
Simply put, MAP_KEY_CHARS is too short. We'll have to make the array larger to avoid this situation (or rework parts of the /f map command).

**Comment by magnusulf** [comment]
Duplicate of #218
